### PR TITLE
Add a disrecommended VPS providers section

### PR DESCRIPTION
**Pull request by szaimen**
See https://help.nextcloud.com/t/help-installing-nc-via-aio-on-vps/153956

Signed-off-by: Simon L <szaimen@e.mail.de>

### CODE REVIEW

**Comment by Zoey2936** [comment]
Maybe mention that KVM / non virtualized servers are recommended?

**Comment by szaimen** [comment]
> Maybe mention that KVM / non virtualized servers are recommended?

Added.

File: readme.md [hunk 1]
```diff
@@ -139,8 +139,10 @@ The easiest way to run it with Portainer on Linux is to use Portainer's stacks f
...
 Although it does not seems like it is the case but from AIO perspective a Cloudflare Tunnel works like a reverse proxy. So please follow the [reverse proxy documentation](./reverse-proxy.md) where is documented how to make it run behind a Cloudflare Tunnel.
 
-### Strato Linux VPS
-Stratos VPS are not recommended for running AIO, since they crash/freeze/make errors when they reach an extremely low PID limit, which is very quickly reached by AIO, see [here](https://github.com/nextcloud/all-in-one/discussions/1747#discussioncomment-4716164), Strato does normally not increase this limit.
+### Disrecommended VPS providers
+- Stratos VPS crash/freeze/make errors when they reach an extremely low PID limit, which is very quickly reached by AIO, see [here](https://github.com/nextcloud/all-in-one/discussions/1747#discussioncomment-4716164), Strato does normally not increase this limit.
+- Hostingers VPS seem to miss a specifc Kernel feature which is required for AIO to run correctly. See [here](https://help.nextcloud.com/t/help-installing-nc-via-aio-on-vps/153956).
+- In return recommended VPS are those that are KVM/non-virtualized as Docker should work best on them.
```
Comment (szaimen) [inline readme.md:145]: WDYT, should we add an additional section about this?

Comment (Zoey2936) [reply readme.md:145]: Yes, would be better…

Comment (szaimen) [reply readme.md]: okay

Comment (szaimen) [reply readme.md]: will do in a follow-up

**Comment by szaimen** [comment]
> Maybe mention that KVM / non virtualized servers are recommended?

See https://github.com/nextcloud/all-in-one/pull/1827

**Review by Zoey2936 — APPROVED** [review]

**Pull request merged**